bindings/core/src/method/utils.rs

```diff
@@ -152,4 +152,7 @@ pub enum UtilsMethod {
         signature: String,
         message: String,
     },
+    /// Creates a UTXOInput from outputId.
+    #[serde(rename_all = "camelCase")]
+    OutputIdToUTXOInput { output_id: String },
```

Comment:
Should this be `OutputIdToUtxoInput`?
I think everywhere else we named it with `txo` small
`GetUtxoChangesById`, `GetUtxoChangesByIndex`, `UtxoInputDto`

Reply:
I changed it, the TS class name is `UTXOInput`, this is why I used it this way.

Reply:
(Please don't resolve comments other people started)